packages/fast-components-class-name-contracts-msft/src/carousel/index.ts

```diff
@@ -67,4 +67,14 @@ export interface CarouselClassNameContract {
      * The dark slide theme
      */
     carousel__themeDark?: string;
+
+    /**
+     * The previous slide transition
+     */
+    carousel__slideTransitionDirectionPrevious?: string;
```

Comment:
In pivot, we have [block]_[el]__animatePrevious

Can we go less verbose and not lose any of the semantics here?

Reply:
The class is added to the overall carousel element, so there's no inner element per se. It could be `carousel__slideAnimatePrevious` I suppose. I think having `slide` enumerated is important, or else the class name becomes too vague.

Reply:
I'm good adding slide into it. Distinction makes sense

Reply:
Will change to `carousel__slideAnimatePrevious` and `carousel__slideAnimateNext` then